examples/routing_service/routing_service_msql_adapter/cpp/RsStreamReader.cxx

```diff
@@ -0,0 +1,134 @@
+/*
+ * (c) 2020 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ *
+ * RTI grants Licensee a license to use, modify, compile, and create derivative
+ * works of the Software.  Licensee has the right to distribute object form
+ * only for use with RTI products.  The Software is provided "as is", with no
+ * warranty of any type, including any warranty for fitness for any purpose.
+ * RTI is under no obligation to maintain or support the Software.  RTI shall
+ * not be liable for any incidental or consequential damages arising out of the
+ * use or inability to use the software.
+ */
+
+#include "RsStreamReader.hpp"
+#include "RsLog.hpp"
+
+#define RTI_RS_LOG_ARGS "RsStreamReader"
+
+using namespace rti::routing::adapter;
+using namespace rti::routing;
+using namespace dds::core::xtypes;
+using namespace dds::sub;
+
+RsStreamReader::RsStreamReader(
+        RsConnection *connection,
+        const StreamInfo &info,
+        const PropertySet &properties,
+        StreamReaderListener *listener)
+        : samplingPeriod_(50),
+          streamInfo_(info.stream_name(), info.type_info().type_name())
+{
+    connection_ = connection;
```

Comment:
you should be using the constructor initializer section instead of setting these here (like you are doing with samplingPeriod_ for example)

Reply:
done